tomls/omnibus-arbitrum-mainnet/spot/tbtc.toml

```diff
@@ -119,4 +115,4 @@ target = ["spotFactory.SpotMarketProxy"]
 fromCall.func = "getMarketOwner"
 fromCall.args = ["<%= extras.synth_btc_market_id %>"]
 func = "setAtomicFixedFee"
-args = ["<%= extras.synth_btc_market_id %>", "<%= parseEther(String(0.01)) %>"]
+args = ["<%= extras.synth_btc_market_id %>", "<%= parseEther(String(0.30)) %>"]
```

Comment:
And here too unexpected change

Reply:
Hey yeah I hopefully explictly made this clear in this pr https://github.com/Synthetixio/SIPs/pull/2087/files